libraries/Matter/examples/MatterWindowCovering/README.md

```diff
@@ -0,0 +1,228 @@
+# Matter Window Covering Example
+
+This example demonstrates how to create a Matter-compatible window covering device using an ESP32 SoC microcontroller. The application showcases Matter commissioning, device control via smart home ecosystems, and manual control using a physical button.
+
+## Supported Targets
+
+| SoC | Wi-Fi | Thread | BLE Commissioning | Status |
+| --- | ---- | ------ | ----------------- | ------ |
+| ESP32 | ✅ | ❌ | ❌ | Fully supported |
+| ESP32-S2 | ✅ | ❌ | ❌ | Fully supported |
+| ESP32-S3 | ✅ | ❌ | ✅ | Fully supported |
+| ESP32-C3 | ✅ | ❌ | ✅ | Fully supported |
+| ESP32-C5 | ✅ | ❌ | ✅ | Fully supported |
+| ESP32-C6 | ✅ | ❌ | ✅ | Fully supported |
+| ESP32-H2 | ❌ | ✅ | ✅ | Supported (Thread only) |
+
+### Note on Commissioning:
```

Comment:
If this is common information for all Matter devices and if it will be included in several examples, please move to a common place and add a reference for it.

Reply:
This is the README.md file that is open with in the Arduino Sketch when the user opens the Arduino IDE with that particular sketch.

It is not possible to create common files beacuse it won't be open in the Arduino IDE.